components/Subscribe/IFTTTItem.js

```diff
@@ -34,88 +38,196 @@ const useStyles = makeStyles(theme => ({
         <CloseIcon />
       </IconButton>
-      <Typography variant="h5">{t`Subscribe to ${platform}`}</Typography>
-      {tutorialYoutubeId ? (
-        <div className={classes.youtube}>
-          <iframe
-            className="iframe"
-            width="100%"
-            height="100%"
-            src={`https://www.youtube.com/embed/${tutorialYoutubeId}`}
-            frameBorder="0"
-            allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
-            allowFullScreen
-          ></iframe>
-        </div>
-      ) : null}
-      <Box
-        display="flex"
-        alignItems="stretch"
-        borderRadius={4}
-        border="1px solid #ddd"
-        marginBottom={2}
-      >
-        <Box
-          display="flex"
-          alignItems="center"
-          p={1}
-          borderRight="1px solid #ddd"
-        >
-          <Typography noWrap variant="body1">{t`Feed URL`}</Typography>
-        </Box>
-        <Box
-          display="flex"
-          alignItems="center"
-          px={1}
-          borderRight="1px solid #ddd"
+      <Box px={3}>
+        <Tabs
+          value={value}
+          onChange={handleChange}
+          aria-label="tabs"
+          variant="fullWidth"
+          indicatorColor="primary"
+          textColor="primary"
         >
-          <TextField
-            defaultValue={feedUrl}
-            InputProps={{ readOnly: true, disableUnderline: true }}
-            onFocus={e => e.target.select()}
-            fullWidth
-          />
-        </Box>
-        <Box display="flex" alignItems="center">
-          <CopyButton
-            content={feedUrl}
-            onSuccess={() => copySuccess(t`Copied to clipboard!`)}
-          >{t`Copy`}</CopyButton>
-        </Box>
+          <CustomTab label="快速訂閱" {...a11yProps(0)} />
+          <CustomTab label="進階設定" {...a11yProps(1)} />
+        </Tabs>
       </Box>
-      <Button
-        variant="contained"
-        color="default"
-        component="a"
-        href={IFTTTAppletUrl}
-        target="_blank"
-        rel="noopener noreferrer"
-        fullWidth
-        size="large"
-      >
-        {t`Subscribe`}
-      </Button>
+      <TabPanel value={value} index={0}>
+        {tutorialYoutubeId ? (
+          <Box py={1}>
+            <div className={classes.youtube}>
+              <iframe
+                className="iframe"
+                width="100%"
+                height="100%"
+                // https://developers.google.com/youtube/player_parameters
+                src={`https://www.youtube.com/embed/${tutorialYoutubeId}?rel=0&modestbranding=1`}
+                frameBorder="0"
+                allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+                allowFullScreen
+              ></iframe>
+            </div>
+          </Box>
+        ) : null}
+        <Box textAlign="center">
+          <Typography variant="h6">{t`訂閱「等你來答」到 ${platform}`}</Typography>
```

Comment:
Wrapping `t` to Chinese characters will cause it to be used as translation keys when we extract translation in the future. Suggest remove `t` from these copies.